server/ntb/publish/ntb_nitf.py

```diff
@@ -263,6 +263,14 @@ def repl_embedded(match):
                 del p_lead.attrib['lede']
             except KeyError:
                 pass
+
+        try:
```

Comment:
it's the ask forgiveness not permission pattern. it's slightly better is the key is supposed to be available.
I used it often by habit, there is not much difference with the other way on such a small thing, but there is nothing wrong by using it.